### PR TITLE
Challenges: Get Present and Future Challenges API Endpoint added

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

**Comment by deshraj** [comment]
@TroJan Can you review this?

File: apps/challenges/views.py [hunk 1]
```diff
@@ -24,7 +24,8 @@
 @authentication_classes((TokenAuthentication,))
 def challenge_list(request, challenge_host_team_pk):
     try:
-        challenge_host_team = ChallengeHostTeam.objects.get(pk=challenge_host_team_pk)
```
Comment (TroJan) [inline apps/challenges/views.py]: why are these changes? autopep8?

File: apps/challenges/views.py [hunk 2]
```diff
@@ -53,7 +54,8 @@ def challenge_list(request, challenge_host_team_pk):
 @authentication_classes((TokenAuthentication,))
 def challenge_detail(request, challenge_host_team_pk, pk):
     try:
-        challenge_host_team = ChallengeHostTeam.objects.get(pk=challenge_host_team_pk)
+        challenge_host_team = ChallengeHostTeam.objects.get(
```
Comment (TroJan) [inline apps/challenges/views.py]: same reason, if this is due to autopep8, then i will prefer that these should go in a seperate pr

File: apps/challenges/views.py [hunk 3]
```diff
@@ -73,7 +75,8 @@ def challenge_detail(request, challenge_host_team_pk, pk):
         if request.method == 'PATCH':
             serializer = ChallengeSerializer(challenge,
                                              data=request.data,
-                                             context={'challenge_host_team': challenge_host_team},
```
Comment (TroJan) [inline apps/challenges/views.py]: autopep8 one

File: apps/challenges/views.py [hunk 4]
```diff
@@ -136,14 +139,26 @@ def disable_challenge(request, pk):
...
     """
-    challenge = Challenge.objects.filter(end_date__lt=timezone.now(), published=True)
-    paginator = PageNumberPagination()
-    paginator.page_size = settings.REST_FRAMEWORK['PAGE_SIZE']
-    result_page = paginator.paginate_queryset(challenge, request)
-    serializer = ChallengeSerializer(result_page, many=True)
-    response_data = serializer.data
-    return paginator.get_paginated_response(response_data)
+    try:
+        if challenge_time.lower() == "past":
```
Comment (TroJan) [inline apps/challenges/views.py]: a better version can be
```
q_params = {'published': True}
if challenge_time.lower() == 'past':
# update q_params accordingly
...

# then use this in queryset like
challenge=Challenge.objects.filter(**q_params)
```

Comment (deshraj) [inline apps/challenges/views.py]: Cool

**Pull request merged**